<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Sanchit-PC" name="com.app.qa.test.ProjectDashboardTest" tests="12" failures="1" timestamp="03 Feb 2019 15:54:58 GMT" time="69.145" errors="0">
  <testcase name="validateLogo" time="5.165" classname="com.app.qa.test.ProjectDashboardTest"/>
  <testcase name="validatePageTitle" time="5.153" classname="com.app.qa.test.ProjectDashboardTest"/>
  <testcase name="validateAccountingStatistics" time="5.856" classname="com.app.qa.test.ProjectDashboardTest"/>
  <testcase name="validateAdministrationStatistics" time="5.868" classname="com.app.qa.test.ProjectDashboardTest"/>
  <testcase name="validateMarketingStatistics" time="6.629" classname="com.app.qa.test.ProjectDashboardTest"/>
  <testcase name="validateSalesStatistics" time="5.768" classname="com.app.qa.test.ProjectDashboardTest"/>
  <testcase name="validateProductionStatistics" time="5.787" classname="com.app.qa.test.ProjectDashboardTest"/>
  <testcase name="validateArchivedStatistics" time="5.747" classname="com.app.qa.test.ProjectDashboardTest"/>
  <testcase name="validateNewStatistics" time="5.825" classname="com.app.qa.test.ProjectDashboardTest"/>
  <testcase name="validateWorkingStatistics" time="5.886" classname="com.app.qa.test.ProjectDashboardTest"/>
  <testcase name="validateDeliveredStatistics" time="5.839" classname="com.app.qa.test.ProjectDashboardTest"/>
  <testcase name="validateStatusUpdateStatistics" time="5.622" classname="com.app.qa.test.ProjectDashboardTest">
    <failure type="java.lang.AssertionError" message="Test Failed - Statistics are not getting updated on updating the status of a record! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Test Failed - Statistics are not getting updated on updating the status of a record! expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at com.app.qa.test.ProjectDashboardTest.validateStatusUpdateStatistics(ProjectDashboardTest.java:102)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- validateStatusUpdateStatistics -->
</testsuite> <!-- com.app.qa.test.ProjectDashboardTest -->
